app/jobs/export_fhir_job.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+# ExportFhirJob: prepare a FHIR export for a client application
+class ExportFhirJob < ApplicationJob
+  queue_as :exports
+  include Sidekiq::Status::Worker
+  sidekiq_options retry: 0
+
+  # Batch size limits number of Patient records details help in memory at once before writing to file.
+  BATCH_SIZE = ENV.fetch('EXPORT_INNER_BATCH_SIZE', 500).to_i unless const_defined?(:BATCH_SIZE)
+  NUM_EXPORTS = 6
+
+  def perform(current_client_application, download, params)
+    patient_ids = Jurisdiction.find_by(id: current_client_application[:jurisdiction_id])&.all_patients_excluding_purged&.pluck(:id)
+    return if patient_ids.nil?
+
+    store transaction_time: DateTime.now.utc
```

Comment:
I was really thrown off by these `store`, `at`, `total` methods at first - but quickly realized it comes from activejob-status. Maybe use parens here, and please also leave a comment saying this is functionality coming from that gem.

Reply:
Done in https://github.com/SaraAlert/SaraAlert/pull/987/commits/5be75cbe252a41cd26747d9b35a057ef7a118ab7.